app/controllers/application_controller.rb

```diff
@@ -11,4 +11,12 @@ def configure_permitted_parameters
     devise_parameter_sanitizer.for(:sign_up) << [:first_name, :last_name, :phone, :time_zone]
     devise_parameter_sanitizer.for(:account_update) << [:first_name, :last_name, :phone, :time_zone]
   end
+
+  def after_sign_in_path_for(resource)
```

Comment:
Unused method argument - `resource`. If it's necessary, use `_` or `_resource` as an argument name to indicate that it won't be used. You can also write as `after_sign_in_path_for(*)` if you want the method to accept any arguments but don't care about them.